src/routes/notifications.route.js

```diff
@@ -0,0 +1,83 @@
+const dbf = require("./../utils/db_conn");
+const {BASE_URL_NOTIFICATION} = require('./../utils/constant')
+const {insertNotification, getNotification} = require("../services/notifications.service");
+
+module.exports = app => {
+    /**
+     * By passing a JWT and profile ID you can get a list of notifications
+     * By passing in the appropriate options, you can search for available inventory in the system
+     * @param user - pass an optional search string to specify the profile ID
+     * for which the notifications will be fetched (only work for admin users)
+     * @param read - set to true to also include seen notifications
+     * @return search results matching criteria || bad input parameter
+     */
+    app.get(BASE_URL_NOTIFICATION,async function (req, res) {
+        let {uuid, read} = req.query
+
+        if (uuid === undefined || uuid === '') {
+            res.send({error: 'bad input parameter'})
+            return
+        }
+
+        if (read === undefined) read = false
+
+        let db;
+
+        await dbf.get(function (_db) {
+            db = _db
+        });
+
+        getNotification(db, {uuid}, (msg) => {
+            console.log(msg)
```

Comment:
use logging package instead of `console.log` (something like [this](https://www.npmjs.com/package/npmlog))